packages/styles/combobox.css

```
@@ -254,3 +254,24 @@
   padding: var(--space-smaller);
   text-align: center;
 }
+
+.Combobox__TextFieldWrapper--multiselect {
+  flex-wrap: wrap;
+  padding-right: var(--space-large);
+}
+
+.Combobox__TextFieldWrapper--multiselect input {
+  width: unset;
+}
```

I would probably recommend setting `width: auto` instead.

```suggestion
.Combobox__TextFieldWrapper--multiselect input {
  width: auto;
}
```

---

Updated